config/configmapprovider/provider.go

```diff
@@ -40,15 +45,27 @@ type Provider interface {
 	// contains the value to be injected in the configuration and the corresponding watcher that
 	// will be used to monitor for updates of the retrieved value.
 	//
-	// watcher callback is called when the config changes. watcher may be called from
+	// `selector` must follow the "<scheme>:<opaque_data>" format. This format is compatible with the URI definition
```

Comment:
Is `<scheme>:` part optional?

Reply:
Currently is not specified, should it be?

Reply:
Done.